server/repositories/cmsQueries/linkPageQuery.js

```diff
@@ -1,14 +1,15 @@
 /* eslint-disable class-methods-use-this */
 const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
 
-class ExternalLinkPageQuery {
+class LinkPageQuery {
   constructor(location) {
     this.location = location;
     this.query = new Query()
-      .addFields('node--external_link', [
+      .addFields('node--link', [
         'title',
         'drupal_internal__nid',
-        'field_external_url',
+        'field_show_interstitial_page',
+        'field_url',
```

Comment:
field has been renamed and rather than object including uri is now just a string